provider/storage_benchmark.py

```diff
@@ -155,6 +157,22 @@ def download_benchmark(self, mode, url, dst, timeout=300):
         self.session_cmd(self.download_cmds[mode].format(dst, url), timeout)
         self.env_files.append(dst)
 
+    def _get_route_ip(self):
+        """ Get the vm route ip address. """
+        route_iface = self.session_cmd("ip route | grep default | awk '{print $5}'").strip()
+        if not route_iface:
+            raise TestError('Failed to get the route table.')
+        return self.session_cmd(
+            "ifconfig %s | grep -E 'inet ' | awk '{print $2}'" % route_iface).strip()
```

Comment:
Function `virt_vm.get_address() ` can do the same thing.

Reply:
Thanks, have updated.